code/modules/mob/living/carbon/human/species/outsider/scp049.dm

```diff
@@ -34,28 +31,6 @@
 	radiation_mod =  0.0                    // No radiation damage
 	flash_mod =      0.0                    // Unflashable
 
-/datum/species/scp049/handle_post_spawn(mob/living/carbon/human/H)
-	. = ..()
-
-/datum/species/scp049/handle_npc(var/mob/living/carbon/human/scp049/H)
```

Comment:
Why are you removing their AI behaviour?

Reply:
This is pretty W.I.P still. Need to replace it, it's very buggy.

Reply:
Completely fair

Reply:
Bay has something called "ai_holder" that I'm trying to delve into. Should be able to help relieve some of the issues it used to have.